build/BuildFrameworksTasks/BuildDesktopVKTask.cs

```diff
@@ -0,0 +1,25 @@
+﻿
+namespace BuildScripts;
+
+[TaskName("Build DesktopVK")]
+public sealed class BuildDesktopVKTask : FrostingTask<BuildContext>
+{
+    // TEMP: Until OSX and Linux is setup to work.
+    public override bool ShouldRun(BuildContext context) => context.IsRunningOnWindows();
+
+    public override void Run(BuildContext context)
+    {        
+        // Generate the native projects.
+        var exit = context.StartProcess("premake5", new ProcessSettings { WorkingDirectory = "src/monogame", Arguments = "clean" });
+        if (exit < 0)
+            throw new Exception($"Setting Premake clean failed! {exit}");
+        exit = context.StartProcess("premake5", new ProcessSettings { WorkingDirectory = "src/monogame", Arguments = "--os=windows --verbose vs2022" });
+        if (exit < 0)
+            throw new Exception($"Setting Premake generation failed! {exit}");
+
+        // Build it.
+        exit = context.StartProcess("msbuild", new ProcessSettings { WorkingDirectory = "src/monogame", Arguments = "monogame.sln /p:Configuration=Release /p:Platform=x64" });
```

Comment:
@harry-cpp - I'm getting the error in the workflow:

> An error occurred when executing task 'Build DesktopVK'.
Error: An error occurred trying to start process '"msbuild"' with working directory 'D:/a/MonoGame/MonoGame/src/monogame'. The system cannot find the file specified.

Any ideas why this is happening?  It seems like that folder clearly exists.

Reply:
Or is it not finding `msbuild`?

Reply:
The issue is that `msbuild` was not installed as part of the workflow.  Fixed it now.